packages/@uppy/companion/src/uppy.js

```diff
@@ -63,7 +63,7 @@ module.exports.app = (options = {}) => {
   app.use((req, res, next) => {
     res.header(
       'Access-Control-Allow-Headers',
-      [res.get('Access-Control-Allow-Headers'), 'uppy-auth-token'].join(', ')
+      [res.get('Access-Control-Allow-Headers'), 'uppy-auth-token', 'uppy-client'].join(', ')
     )
```

Comment:
uppy-versions?

Reply:
`Uppy-Versions`

Reply:
Node.js will lowercase it for us, but i think so far we've tried to use uppercasing like that in tus/uppy headers